pandas/core/window.py

```diff
@@ -1354,7 +1354,9 @@ def _get_center_of_mass(com, span, halflife, alpha):
 
 
 def _offset(window, center):
-    if not com.is_integer(window):
+    if com.is_float(window):
```

Comment:
this needs to raise if its not an integer for Rolling and sub-classes, or int/array of ints for Window